flutter/example/integration_test/integration_test.dart

```diff
@@ -18,7 +18,6 @@ void main() {
         bundle: SentryAssetBundle(enableStructuredDataTracing: true),
         child: const MyApp(),
       )));
-      await tester.pumpAndSettle();
```

Comment:
Isn't that needed anymore?

Reply:
Does not make a difference in the tests as far as i could tell.